### PR TITLE
feat: add nuxt-umami

### LINKED ISSUE #421: Add 'nuxt-umami'
Hi,
I created a wrapper for [Umami Analytics](https://umami.is) compatible with Nuxt 3.

It is a rewrite of [nuxt-umami-module](https://github.com/joe-pritchard/nuxt-umami-module) but for Nuxt 3 (and theoretically, Nuxt 2).
`nuxt-umami-module` has an SSR issue, and I don't think it is being maintained anymore. I also tried contacting Joe Pritchard but got no response.

Repo: https://github.com/ijkml/nuxt-umami
npm: https://www.npmjs.com/package/nuxt-umami

I would appreciate any help/feedback!

### PR DESCRIPTION
**Pull request by ijkml**
Add nuxt-umami module, a wrapper for Umami Analytics

Resolves #421

### CODE REVIEW

**Review by Atinux — APPROVED** [review]

**Comment by ijkml** [comment]
Hi @pi0 @Atinux 
It's been a while and I haven't heard back, I know you've been really busy with Nuxt. So, I was wondering, is there anything I could do to help approve/push  this PR?

File: modules/umami.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+github: https://github.com/ijkml/nuxt-umami
+website: https://github.com/ijkml/nuxt-umami
+learn_more: https://umami.is/docs/
+category: Analytics
+type: 3rd-party
+maintainers:
+  - name: ML
+    github: ijkml
+compatibility:
+  nuxt: ^2.0.0 || ^3.0.0
```
Comment (pi0) [inline modules/umami.yml:15]:
```suggestion
  nuxt: ^2.0.0 || >=3.0.0.rc.6
```

**Review by pi0 — APPROVED** [review]
Nice module! Sorry for late review. I've added some comments in https://github.com/ijkml/nuxt-umami/issues/3 (rc.9+ requires a change) but looks go for me to list.

**Comment by ijkml** [comment]
Thanks for your all your help with improving the module :rocket: (https://github.com/ijkml/nuxt-umami/issues/3)

I have merged and released v1.0.0 of the module, please let me know if this is good to go.
https://github.com/ijkml/nuxt-umami/releases/tag/v1.0.0

**Pull request merged**